ci/appveyor/run-with-visual-studio.cmd

```diff
@@ -0,0 +1,88 @@
+:: To build extensions for 64 bit Python 3, we need to configure environment
```

Comment:
Found the source.
https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor/run_with_env.cmd

Will add a comment and push an update, soon.